.eslintrc.json

```diff
@@ -2,7 +2,8 @@
   "env": {
     "browser": true,
     "es6": true,
-    "node": true
+    "node": true,
+    "jest": true
```

Comment:
Sorry my bad, I didn't notice this the last time. As I mentioned in the first review, you can use overrides in the eslint. So you can specify rules for specific folders/files. With this you can say that only the `.test.js` files have the jest environment. If you don't do this, then eslint will let us use the describe global method in any source file which of course not exists.

Reply:
fixed